### ISSUE TITLE
Pigment epithelium detachment

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10868]
**Preferred term label:**
Pigment epithelium detachment

**Synonyms**
PED

**Definition (free text, please give PubMed ID)**
Pigment Epithelial Detachment (PED) refers to the separation of the retinal pigment epithelium (RPE) from the underlying Bruch’s membrane.

**Parent term (use hpo.jax.org/app)**
7703 (not sure, though)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
	•	Wet (Neovascular) AMD: PEDs are often associated with choroidal neovascularization (CNV). Subretinal fluid, exudates, or hemorrhage may accompany the PED.

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001300
- label: Pigment epithelium detachment
- parents: [Abnormality of retinal pigmentation (HP:0007703)](https://hpo.jax.org/app/browse/term/HP:0007703)
- pmid: PMID:28280989
- def: Separation of the retinal pigment epithelium (RPE) from the underlying Bruch membrane.
- comment: Pigment epithelial detachment (PED) is a common feature in patients with exudative age-related macular degeneration. The prevalence of a PED has been reported to be 10-64% in eyes with polypoidal choroidal vasculopathy, and 10-22% in eyes with typical exudative AMD with choroidal neovascularization. The natural course is characterised by enlargement of the PED with consecutive regression and atrophy of the RPE and outer retina in avascular PEDs or formation of a disciform scar in vascularized PEDs.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10868
- synonyms:
